src/datatypes/User.ts

```diff
@@ -17,8 +17,8 @@ export default interface User {
   about?: string;
   hideTagColors?: boolean;
   followedCubes?: string[];
-  followedUsers?: string[];
-  following?: string[];
+  followedUsers?: string[]; //Who this user is following
```

Comment:
Comments mainly because I confused myself when looking at the dynamo results directly lol